frontend/www/js/omegaup/components/user/Language.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <li class="dropdown">
+    <a class="dropdown-toggle"
+        data-toggle="dropdown"
+        href="#"><img v-bind:src="flagUrl"> <span class="caret"></span></a>
```

Comment:
no puedes hacer algo así como

```suggestion
        href="#"><img v-bind:src="showFlagUrl(selectedLanguage)"> <span class="caret"></span></a>
```

?

Reply:
Listo, también se elimina la propiedad computada